Gemfile

```diff
@@ -8,6 +8,13 @@ group :development do
   gem 'pry'
 end
 
+if RUBY_VERSION < "2.0"
+  gem 'json', '< 2.0.0'
```

Comment:
I don't think it should be this library responisbility to make people use the latest working version (maybe somebody has some other reaosons to use 1.7.x). The reason of this contaraits are really to note which versions are known not working on a given RUBY_VERSION. Also, what if 1.9.x version gets released (here it's just a theory, but still), I think '< 2.0.0' serves better for this purposes here.